common/styles/base/_inline_fonts.scss

```diff
@@ -1,7 +1,8 @@
-  unicode-range: U+44, U+46, U+58-59, U+65-90, U+97-122;
+  unicode-range: U+0020, U+0027, U+002C, U+002E, U+0030-0039, U+003F, U+0041-005A, U+0061-007A;
```

Comment:
Could you add a comment listing what these characters are, and maybe a link to the resource you used for looking them up?

Reply:
I'm now wondering whether it would be better to leave the unicode-range out all together (seeing as it is always used) or leave it there so we know what is in the subset?